pkg/crc/validation/validation.go

```diff
@@ -2,11 +2,11 @@ package validation
 
 import (
 	"fmt"
+	"github.com/code-ready/crc/pkg/crc/version"
```

Comment:
shouldn't this go after the `crc/machine` import?

Reply:
@cfergeau No, we want to use `crc/machine` only for machine related stuff, this is independent of it.

Reply:
I'm only saying that the `"github.com/code-ready/crc/pkg/crc/version"` line should be moved down, I'm not suggesting moving code in other packages.